geonetwork/catalogue_aodn_upgrade/README-Validation.md

```diff
@@ -0,0 +1,108 @@
+[bluenet_marvlis_uuids.txt](https://github.com/aodn/utilities/files/6952457/bluenet_marvlis_uuids.txt)
+[bluenet_marvlis_uuids.txt](https://github.com/aodn/utilities/files/6952460/bluenet_marvlis_uuids.txt)
+# Catalogue AODN hosted MCP records
+
+- Create a fresh working directory in a convenient location
+
+```
+mkdir /tmp/catalogue_aodn_hosted_mcp
+```
+
+- OPTIONAL: Obtain a list of UUIDS for non harvested mcp records that need correction
+
+This step is only required if a full validation is required and the records to be corrected are unknown.
+
+```
+ssh 6-aws-syd "sudo -u postgres psql -d geonetwork_aodn -c \"\\copy (select uuid from metadata where isharvested = 'n' and istemplate = 'n' and (schemaid = 'iso19139.mcp' or schemaid = 'iso19139.mcp-1.4' or schemaid = 'iso19139.mcp-2.0')) to stdout\" > catalogue_aodn_non_harvested_mcp_uuids.txt"
+scp 6-aws-syd:catalogue_aodn_non_harvested_mcp_uuids.txt /tmp/uuids.txt
+```
+
+- Download the MEFS
+
+Use this list of uuids ([corrected-uuids.txt](https://github.com/aodn/utilities/files/6951934/corrected-uuids.txt)
+) or the one generated in the previous step to download the MEFS to be validated.
+
+```
+./download_mefs.sh /tmp/uuids.txt /tmp/catalogue_aodn_hosted_mcp
+```
+
+- Locate misplaced info.xml files
+
+Some info.xml files are in an unexpected location for the validator.  Find and move these.
+
+```
+for f in $(find /tmp/catalogue_aodn_hosted_mcp -name info.xml | grep metadata); do mv $f $(dirname $(dirname $f)); done
+```
+
+- Source and copy the schema plugins to one location
+
+These can be obtained from https://github.com/aodn/schema-plugins/tree/180e374a2f1ac181a79bc3b9432b46739301e0e6
```

Comment:
what's the significance of https://github.com/aodn/schema-plugins/tree/180e374a2f1ac181a79bc3b9432b46739301e0e6
i.e. rather than the latest master version of this repo?